htdocs/install/mysql/tables/llx_expeditiondet_dispatch.sql

```diff
@@ -0,0 +1,29 @@
+-- ===================================================================
```

Comment:
Why adding a table llx_expeditiondet_dispatch
If we look how reception is done, we have a table
llx_reception and a table llx_receptiondet (where name was historically named llx_commande_fournisseur_dispatch but name llx_receptiondet would have been better).

So for shipment we should have:
llx_expedition and llx_expeditiondet, and we alreaydy have these tables. For the detail of batch, on supplier side, it is in same table llx_receptiondet=llx_commande_fournisseur_dispatch, for shipment, we have the table
llx_expeditiondet_batch.
So i don't see reason to add another table.